lib/create-app.js

```diff
@@ -2,14 +2,15 @@ const copy = require('kopy');
 const path = require('path');
 const kebabCase = require('just-kebab-case');
 const pascalCase = require('just-pascal-case');
+const cliPackageJSON = require('../package.json');
```

Comment:
what for to import the whole package.json?
you can for example destructure the version, like:
`const { version } = require('../package.json');`